app/model/db/idx_token.py

```diff
@@ -0,0 +1,356 @@
+"""
+Copyright BOOSTRY Co., Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+"""
+from typing import Union, Type
+from sqlalchemy import (
+    Column,
+    String,
+    BigInteger,
+    Boolean,
+    Float,
+    Text,
+    JSON,
+    DateTime
+)
+from app.model.db import Base
+
+
+class TokenBase(Base):
+    """Token Base Attribute"""
+    __abstract__ = True
+
+    # Token Address
+    token_address = Column(String(42), primary_key=True)
+    # Token Template
+    token_template = Column(String(40))
+    # Owner Address
+    owner_address = Column(String(42), index=True)
+    # Company Name(Corporate)
+    # NOTE: Short-term cache required
+    company_name = Column(Text)
+    # RSA Public Key(Corporate)
+    # NOTE: Short-term cache required
+    rsa_publickey = Column(String(2000))
+    # Name
+    name = Column(String(200))
+    # Symbol
+    symbol = Column(String(200))
+    # Total Supply
+    # NOTE: Short-term cache required
+    total_supply = Column(BigInteger)
+    # Tradable Exchange
+    tradable_exchange = Column(String(42), index=True)
+    # Contact Information
+    contact_information = Column(String(2000))
+    # Privacy Policy
+    privacy_policy = Column(String(5000))
+    # Status
+    # NOTE: Short-term cache required
+    status = Column(Boolean)
+    # Max Holding Quantity
+    max_holding_quantity = Column(BigInteger)
+    # Max Sell Amount
+    max_sell_amount = Column(BigInteger)
+    # Cached time of short-term cache
+    short_term_cache_created = Column(DateTime)
+    # Last synced block number of token attribute events
+    latest_block_number = Column(BigInteger, default=-1)
```

Comment:
Added two column.
- ```short_term_cache_created```: column for attributes which expire earlier.
- ```latest_block_number```: column for last synced block number